recipe/meta.yaml

```diff
@@ -51,8 +57,8 @@ test:
     - multipart
   commands:
     - pip check
-    - langsmith -h
```

Comment:
We may benefit from keeping this but just comment out. Then add another comment above to test with langchain.